app/src/androidTest/java/com/android/voyageur/ui/trip/TobTabsTest.kt

```diff
@@ -74,4 +73,36 @@ class TopTabsTest {
     composeTestRule.onNodeWithText("Settings").assertIsSelected()
     composeTestRule.onNodeWithTag("settingsScreen").assertIsDisplayed()
   }
+
```

Comment:
Here, I think that you could also implement a test which simulates opening addActivity, closing it/creating an activity, and checking if you return to the correct screen. Let me know your opinion!

Reply:
Thanks for your suggestion! Since that test requires more screens, it would rather be and end-to-end test. We can do that in a future PR